week-3/todo-list/index.html

```diff
@@ -3,25 +3,38 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Todo List app</title>
     <link rel="stylesheet" href="style.css" />
+    <link
```

Comment:
In the end I don't think you used a custom font, so this `<link>` tag can be removed